packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -212,6 +212,9 @@ export const styles = (theme: ThemeType): JssStyles => ({
     marginLeft: 'auto',
     marginRight: 'auto',
   },
+  postBody: {
+    width: 'max-content',
+  },
```

Comment:
Recommendations loading would previously cause this column to go to its full widths, so now I am making it always be full width